packages/plugin-ext/doc/how-to-add-new-custom-plugin-api.md

```diff
@@ -5,18 +5,24 @@ That will require API that goes beyond what's in the VS Code Extension API and t
 You can do that by implementing a Theia extension that creates and exposes an API object within the plugin host.
 The API object can be imported by your plugins and exposes one or more API namespaces.
 
-Depending on the plugin host we can either provide a frontend or backend plugin API:
+Depending on the plugin host we can either provide a frontend or backend plugin API, or an API for headless plugins that extend backend services that are not tied to any frontend connection:
```

Comment:
Is "extend" really the correct term here? A plugin may extend backend services or it may simply use them to perform whatever it needs to do. Maybe a more neutral term such as "has access to" or something like that may be more fitting?

Reply:
I suppose a plugin could be deployed that is just activated on start-up and then only calls into Theia's APIs, never actually "extending" some service. But even that is an extension (qua expansion) of Theia's run-time behaviour. This is an _extension_ mechanism, after all. 😝 

I'll try to revise for clarity.